python/ray/autoscaler/updater.py

```diff
@@ -258,9 +258,8 @@ def do_update(self):
                             for cmd in self.initialization_commands:
                                 self.cmd_runner.run(
                                     cmd,
-                                    ssh_options_override=SSHOptions(
-                                        self.auth_config.get(
-                                            "ssh_private_key")))
+                                    ssh_options_override_ssh_key=self.
+                                    auth_config.get("ssh_private_key"))
```

Comment:
Can you throw a comment here explaining that we are setting this key to override SSH Options to not use the persistent SSH connection.